src/main/java/net/reini/rabbitmq/cdi/ConnectionManager.java

```diff
@@ -0,0 +1,208 @@
+package net.reini.rabbitmq.cdi;
+
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import java.io.IOException;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeoutException;
+import java.util.concurrent.locks.Condition;
+import java.util.concurrent.locks.ReentrantLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * <p>
+ * Manager manages one connection to one broker. The Manager will reconnect in case the connection is lost, and keeps constantly checking the connection status.
+ * </p>
+ *
+ * @author Patrick Reinhart
+ */
+public class ConnectionManager {
+
```

Comment:
My desired ordering of instance variables (having a black line in between but no empty line after the class definition):
static final
final
non final
non final volatile

Reply:
fixed